numpy/core/tests/test_scalarmath.py

```diff
@@ -150,29 +156,87 @@ def test_divmod_basic(self):
                 msg = fmt % (dt1, dt2, sg1, sg2)
                 a = np.array(sg1*71, dtype=dt1)[()]
                 b = np.array(sg2*19, dtype=dt2)[()]
-                div, rem = divmod(a, b)
+                div = self.floordiv(a, b)
+                rem = self.mod(a, b)
                 assert_allclose(div*b + rem, a, err_msg=msg)
```

Comment:
Exact. Hmm, looks redundant with other tests of exactness, should probably be removed.